File: #1051.height-checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {

        int size = heights.size();
        vector<int> v = heights;
        std::sort(v.begin(), v.end());
        int num = 0;
        for (int i = 0; i < size; i++) {
            if (heights[i] != v[i]) {
                num++;
            }
        }
        return num;
    }
};

File: #1051.height-checker.js
/**
 * @param {number[]} heights
 * @return {number}
 */
var heightChecker = function(heights) {
    let count = 0;
    let sorted = [...heights].sort((left, right) => left - right);
    for (let i = 0; i < heights.length; i++) {
        if (heights[i] !== sorted[i]) {
            count++;
        }
    } 
    return count;
};
